crates/bevy_hierarchy/README.md

```diff
@@ -0,0 +1,3 @@
+# Bevy Hierarchy
+
+Provides the ability to work with a parent-child hierarchy of entities.
```

Comment:
This should probably be expanded somewhat to describe how it propagates transforms down the tree.